pom.xml

```diff
@@ -33,7 +33,7 @@
     Note: when upgrading the Quarkus version or the DataStax Java driver version, make sure that you
     upgrade them in the quickstart module too.
     -->
-    <quarkus.version>3.0.0.Final</quarkus.version>
+    <quarkus.version>3.1.0.CR1</quarkus.version>
```

Comment:
Let's revert this change; I'd like to keep this PR focused on the RCBI changes only.